Examples/UseCases.ipynb

```diff
@@ -82,10 +92,16 @@
     }
    ],
    "source": [
-    "from pydkpro import Pipeline, Component  \n",
+    "from pydkpro import Pipeline, Component\n",
+    "\n",
     "p = Pipeline(version=\"2.0.0\", language='en')\n",
     "p.add(Component().clearNlpSegmenter())\n",
+    "\n",
+    "// REC: Is printTagSet not false by default?\n",
     "p.add(Component().stanfordPosTagger(variant='fast-caseless', printTagSet='false'))\n",
+    "\n",
+    "// If it has a build method, maybe it should be a PipelineBuilder? Also, the build method should return something,\n",
+    "// i.e. the actual pipeline which maybe can later be used to shut the pipeline down?\n",
```

Comment:
I am not sure about this comment. Could you please elaborate on it.

Reply:
Cf. https://dzone.com/articles/design-patterns-the-builder-pattern

If something has a `build()` method, it looks like a builder pattern. A builder does not return itself in the `build()` method but rather the object it was used to construct. In your case, the code looks like you want to have a class `PipelineBuilder` whose `build()` method eventually produces the actual readily-configured `Pipeline` which then can be used to process stuff.

Reply:
Thanks for this pattern. Yes, as soon as the backend gets ready, we will update like this.